versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
```

Comment:
```suggestion
  required_version = ">= 0.12.0"
```

please also remove the upper limit for TF version, we don't use upper limit anymore

Reply:
Sure, done in bd4d0fe